### PR TITLE
Angular / Vue , socialProviders, signUpAttributes documentation fix

### PR DESCRIPTION
**Pull request by ErikCH**
*Issue #, if available:*

Fixed the following issues
 

- [x] Added `socialProviders` to Angular and Vue
- [x] Added documentation for `signUpAttributes` for Angular and Vue
- [x] Fixed issue on documentation site where clicking framework button twice caused a `null` framework to display

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/src/components/Layout/FrameworkChooser.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ export const FrameworkChooser = ({ platform }) => {
       size="small"
       style={{ margin: '0 auto' }}
       onChange={(value: string) => {
+        if (!value) return;
```
Comment (ErikCH) [inline docs/src/components/Layout/FrameworkChooser.tsx:26]: If social providers is null, it won't try to switch to a `null` framework.

File: docs/src/pages/components/authenticator/signUpAttributes.all.angular.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    'name',
+    'nickname',
+    'phone_number',
+    'picture',
+    'preferred_username',
+    'profile',
+    'updated_at',
+    'website',
+    'zoneinfo', ]"
+    ></amplify-authenticator>
```
Comment (wlee221) [inline docs/src/pages/components/authenticator/signUpAttributes.all.angular.mdx:20]: Minor style nit: we can deindent this and match the prettier style.

```suggestion
<amplify-authenticator 
  [signUpAttributes]="[
    'address',
    'birthdate',
    'email',
    'family_name',
    'gender',
    'given_name',
    'locale',
    'middle_name',
    'name',
    'nickname',
    'phone_number',
    'picture',
    'preferred_username',
    'profile',
    'updated_at',
    'website',
    'zoneinfo', 
  ]"
></amplify-authenticator>

```

File: docs/src/pages/components/authenticator/signUpAttributes.all.vue.mdx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    'name',
+    'nickname',
+    'phone_number',
+    'picture',
+    'preferred_username',
+    'profile',
+    'updated_at',
+    'website',
+    'zoneinfo', ]"
+    />
```
Comment (wlee221) [inline docs/src/pages/components/authenticator/signUpAttributes.all.vue.mdx:21]: Similarly, 
```suggestion
    'zoneinfo',
  ]"/>
```

**Review by wlee221 — APPROVED** [review]
LGTM, just one non-blocking style nit!

File: docs/src/pages/components/authenticator/signUpAttributes.all.angular.mdx (hunk 2 above)
Comment (ErikCH) [reply docs/src/pages/components/authenticator/signUpAttributes.all.angular.mdx:20]: OK I see. I'll get this fix in my next PR.

**Pull request merged**